static/src/javascripts/projects/common/utils/config.js

```diff
@@ -19,6 +19,10 @@ define([
         config.page.adUnit = ['/', config.page.dfpAccountId, '/', adUnitOverride].join('');
     }
 
```

Comment:
Well, it is implicit since it utils/config is a dependency of dfp.js. 

However, maybe we should set it so that first one there, keeps it.